code/model/Subsite.php

```diff
@@ -422,11 +480,11 @@ public static function writeHostMap($file = null)
         }
 
         if (!$file) {
-            $file = Director::baseFolder().'/subsites/host-map.php';
+            $file = Director::baseFolder() . '/subsites/host-map.php';
```

Comment:
We could replace this with the module manifest now - i.e. `$file = ModuleLoader::inst()->getModule('silverstripe/subsites')->getRelativeResourcePath('host-map.php')`